server/backup_runner.go

```diff
@@ -149,6 +149,15 @@ func (s *BackupRunner) scheduleBackupTask(ctx context.Context, database *api.Dat
 		return fmt.Errorf("failed to get migration history for database %q: %w", database.Name, err)
 	}
 
+	// Return early if the backup already exists.
+	backup, err := s.server.BackupService.FindBackup(ctx, &api.BackupFind{Name: &backupName})
+	if err != nil {
+		return fmt.Errorf("failed to find backup %q, error %v", backupName, err)
+	}
+	if backup != nil {
+		return nil
```

Comment:
Return common.Conflict and let caller handle instead?

Reply:
Since we return nil in line 173/174, we can return nil here consistently. This change avoid to making postgres log spammy already exists warnings.